lib/spack/spack/database.py

```diff
@@ -1247,10 +1247,14 @@ def _query(
         return results
 
     def query_local(self, *args, **kwargs):
+        """Query only the local Spack database."""
         with self.read_transaction():
             return sorted(self._query(*args, **kwargs))
 
+    query_local.__doc__ += _query.__doc__[_query.__doc__.index('\n'):]
```

Comment:
@becker33 can you extract the common doc into a string vs. indexing into the docstring of `_query`?